### ISSUE TITLE
JS API

### ISSUE THREAD

**Issue by jonathanong** [less/less-docs#212]
http://lesscss.org/usage/

there's no docs on the JS API. can't build anything with less wityhout it

**Comment by christophercliff** [comment]
Are you accepting PRs for this? Is there an appropriate location for it?

**Comment by SomMeri** [comment]
@christophercliff yes, we will gladly accept pull request for documentation, thank you in advance :) My first guess would be to put it [here](https://github.com/less/less-docs/blob/master/content/usage/programmatic-usage.md), but do not take it as a dogma.

**Comment by lukeapage** [comment]
We had a vague plan to document with jsdoc and auto generate but im not
sure how viable that is with our project structure. It wouldnt stop us
merging any prs though.
